src/Kros.AspNetCore/README.md

```diff
@@ -28,30 +28,84 @@ Menný priestor `Kros.AspNetCore.Middlewares` obsahuje užitočné middlewares,
 
 V adresári `Extensions` sa nachádzajú rôzne rozšírenia štandardných tried (rozhraní) v Asp.Net Core, ktoré zjednodušujú prácu s nimi.
 
-- ### ConfigurationExtensions
-
-  Jednoduchšie získavanie jednotlivých nastavení z konfigurácie.
-
-- ### DistributedCacheExtensions
-
-  Jednoduchšie vkládanie/získavanie komplexných typov z/do distribuovanej keše.
+### Configuration
+
+Na jednoduchú konfiguráciu slúži extension metóda `ConfigureOptions<TOptions>(Configuration)`. Táto metóda triedu `TOptions`
+napojí na nastavenia (konfigurácia napríklad v `appsettings.json`) podľa názvu triedy a triedu zaregistruje do DI kontajnera ako:
+
+- `IOptions<TOptions>`
+- Singleton `TOptions`
```

Comment:
If we register that as a singleton, for someone can be strange when he register provider with `reloadOnChange` `AddJsonFile("appsettings.json", optional: false, reloadOnChange: true);`.  He can expect to receive new values.

We may explicitly write into readme that this will not work properly when argument `reloadOnChange` is set to `true`.

Reply:
If you want to use [config reloading](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-3.0#reload-configuration-data-with-ioptionssnapshot), you have to use `IOptionsSnapshot<TOptions>` instead of `IOptions<TOptions>`. So this was never working with our extensions.

Reply:
Hot reload is supported now.